src/app/utils/render.js

```diff
@@ -0,0 +1,18 @@
+const render = (parentId, ...children) => {
+  if (!children || !parentId) {
+    throw new Error(
+      'In params must have been added parentId and at least one child to render in it',
+    );
+  }
```

Comment:
@DominikNowak please fix the rest operator issue as we talked before. Let me know when ready. Then you've got my approval 🚀

Reply:
done :)